pkg/config/config.go

```diff
@@ -586,6 +589,34 @@ func InitializeTestingConfig() {
 	SetDebugMode(true)
 	SetDataPath("data/")
 }
+func ReadRunModConfig() bool {
+	runModFilePath := "data/common/runmod.cfg"
+	if _, err := os.Stat(runModFilePath); err == nil {
+		jsonData, err := os.ReadFile(runModFilePath)
+		if err != nil {
+			return false
+		}
+		var runModConfig struct {
+			PQSConfig string `json:"PQSConfig"`
+		}
```

Comment:
define this struct at the top of this file. Also don't create a new name, we already have a `pqsEnabled` reuse that name